benchmarks/runners/apfel_.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 """
```

Comment:
It looks like the formatter from the pre-commit hooks deleted the enconding (?). This is not that much needed since python>=3 but maybe for the consistency with the other files we should keep it.

Reply:
We're just incrementally removing them, with `pre-commit`, in order to avoid a large diff.

As the formatter explains, that is a bunch of useless characters by now, so we will remove. But if you do all at once, you generate a large diff with a lot of files changed. So there are two options: either we do it all at once in a single commit, or we just update files changed. `pre-commit` is already doing by default the second, so we thought there was no need for the first :)

Reply:
I agree then that removing it for the changed files would be fine, and if at some point one would like to remove it in all the files then one can do so in a single commit (the number of files changed then will be much smaller).

Please ignore the above comment then!